client_firmware/client_firmware.ino

```diff
@@ -1,17 +1,18 @@
 #include <EEPROM.h>
 #include <EtherCard.h>
-#include <IPAddress.h>
 #include <avr/wdt.h>
 
 #define CSPIN 10
 // NOTE: All of these timers are in Milliseconds!
-// Ping our target every 2 seconds
-#define PINGER_INTERVAL 2000
+// Ping our target every 60 seconds
+#define PINGER_INTERVAL 60000
```

Comment:
I didn't notice battery drain; I haven't run the code long enough to tell yet. This was a pro-active change.

Still, if iPhones behave that way, then yes it makes sense to revert that part.

BTW, I'm going to experiment with enabling multicast reception sometime. That'll e.g. pick up the packets my phone sends to query the LAN for ChromeCasts. I expect any device using MDNS, SSDP, UPnP or similar would end up benefiting from that. Perhaps then, I'll make the ping sending only happen if another packet type hasn't been since within the ping timeout, so pings don't happen if the device is active?